docs/markdown/Python/python-goals/python-test-goal.md

```diff
@@ -553,13 +553,15 @@ You can change the output dir with the `output_dir` option in the `[coverage-py]
 
 You may want to set `[coverage-py].fail_under` to cause Pants to gracefully fail if coverage is too low, e.g. `fail_under = 70`.
 
-You may use a Coverage config file, e.g. `.coveragerc` or `pyproject.toml`. Pants will autodiscover the config file for you, and you can also set `[coverage-py].config` in your `pants.toml` to point to a non-standard location. You must include `relative_files = true` in the `[run]` section for Pants to work.
-
-```text .coveragerc
-[run]
-relative_files = true
-branch = true
-```
+You may use a Coverage config file, e.g. `.coveragerc` or `pyproject.toml`. Pants will autodiscover the config file for you, and you can also set `[coverage-py].config` in your `pants.toml` to point to a non-standard location.
+
+> 📘 You must include `relative_files = true` in the `[run]` section for Pants to work:
```

Comment:
Github's diff view makes this somewhat non-obvious. Is the only change here moving this sentence and code block to be a more obvious callout?

Reply:
yup